addons/mail/static/src/core/thread_service.js

```diff
@@ -1082,22 +1083,27 @@ export class ThreadService {
         if (!persona) {
             return DEFAULT_AVATAR;
         }
+        let avatarCacheKey = "";
+        if (persona.write_date) {
+            avatarCacheKey = `?unique=${persona.write_date}`;
+        }
         if (persona.is_company === undefined && this.store.self?.user?.isInternalUser) {
             this.personaService.fetchIsCompany(persona);
         }
         if (thread?.model === "discuss.channel") {
             if (persona.type === "partner") {
-                return url(`/discuss/channel/${thread.id}/partner/${persona.id}/avatar_128`);
+                return url(`/discuss/channel/${thread.id}/partner/${persona.id}/avatar_128${avatarCacheKey}`);
```

Comment:
should use 2nd param of `url` method rather than the route to pass extra params

Reply:
Right ! Done :)